dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -2396,8 +2516,8 @@ data:
           {
             "current": {
               "selected": true,
-              "text": "rhtap-observatorium-production",
-              "value": "rhtap-observatorium-production"
+              "text": "rhtap-observatorium-stage",
+              "value": "rhtap-observatorium-stage"
```

Comment:
Please refrain from changing the default selected datasource.

Reply:
Oops, I didn't realize that had changed.  I've fixed it.